src/renderer/root/components/App/Toasts/Toast/Toast.scss

```diff
@@ -0,0 +1,35 @@
+.toast {
+  @include fade-in;
```

Comment:
```
cursor: default 
```
```
.noselect {
  -webkit-touch-callout: none; /* iOS Safari */
    -webkit-user-select: none; /* Safari */
     -khtml-user-select: none; /* Konqueror HTML */
       -moz-user-select: none; /* Firefox */
        -ms-user-select: none; /* Internet Explorer/Edge */
            user-select: none; /* Non-prefixed version, currently
                                  supported by Chrome and Opera */
}
```
This could be applied to more things (like browser tabs, login/register tabs or such)
In my opinion it is way better :P Feels like a solid "object" instead of a box with some text

Reply:
Yeah, I have a separate branch to apply the `user-select` and `cursor` globally.  I meant to get a PR up for it this past weekend, but didn't get to wrap it up.  I'm planning to get a separate PR up for it though. 😄

Reply:
Just opened #417 for this.